doc/classes/Font.xml

```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		Font contains a Unicode-compatible character set, as well as the ability to draw it with variable width, ascent, descent and kerning. For creating fonts from TTF files (or other font formats), see the editor support for fonts.
+		[b]Note:[/b] If a font doesn't contain a character used in a string, the character in question will be hidden without displaying any replacement character in the string.
```

Comment:
`BitmapFont` will display nothing.
`DynamicFont` will try to display 0xFFFD if it is in font.

Reply:
> `DynamicFont` will try to display 0xFFFD if it is in font.

Weird, I thought that wasn't the case. It seems nobody can get a fallback character to display in practice. Should we change the character displayed or try another fallback character if `0xFFFD` isn't available?

Reply:
Test project in the #40405 do display replacement character (diamond with a question mark), it just has Bold and Regular fonts with the different characters supported.

Reply:
So should this change be updated to mention that a display replacement character will be shown *if supported by the font*?

Reply:
Ping @Calinou @bruvzg.

Reply:
With `BitmapFont` nothing will be displayed even if the `U+FFFD` character is supported by the font.